docs/reference/modules/aggregations/matrix/stats.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[modules-matrix-aggregations-stats]]
+=== Matrix Stats
+
+The `matrix_stats` aggregation is a numeric aggregation that computes the following statistics over a set of document fields:
+
+[horizontal]
+`count`:: Number of per field samples included in the calculation.
+`mean`:: The average value for each field.
+`variance`:: Per field Measurement for how spread out the samples are from the mean.
+`skewness`:: Per field measurement quantifying the asymmetric distribution around the mean.
+`kurtosis`:: Per field measurement quantifying the shape of the distribution.
+`covariance`:: A matrix that quantitatively describes how changes in one field are associated with another.
+`correlation`:: The covariance matrix scaled to a range of -1 to 1, inclusive. Describes the relationship between field
+            distributions.
+
+The following example demonstrates the use of matrix stats to describe the relationship between income and poverty.
+
+[source,js]
+--------------------------------------------------
+{
+    "aggs": {
+        "matrixstats": {
+            "matrix_stats": {
+                "field": ["poverty", "income"]
+            }
+        }
+    }
+}
+--------------------------------------------------
+
+The aggregation type is `matrix_stats` and the `field` setting defines the set of fields (as an array) for computing
+the statistics. The above request returns the following response:
+
+[source,js]
+--------------------------------------------------
+{
+    ...
+    "aggregations": {
+        "matrixstats": {
+            "field": ["income", "poverty"],
```

Comment:
other APIs that take several fields such as the `query_string` query take `fields` rather than `field`, should we be consistent here?